Server/Forms/FrmRegistryEditor.cs

```diff
@@ -35,6 +35,16 @@ public FrmRegistryEditor(Client c)
             InitializeComponent();
         }
 
+        protected override CreateParams CreateParams
```

Comment:
Applying the style to the form is beneficial to all child controls, see here: http://stackoverflow.com/a/25878263